pandas/core/frame.py

```diff
@@ -5405,6 +5405,94 @@ def _construct_result(self, result) -> "DataFrame":
         out.columns = self.columns
         return out
 
+    @Appender(
```

Comment:
is this added to the doc-string tests?

Reply:
I can't seem to find a doc-string test module, is there an example to refer to?

Reply:
See `ci/code_checks.sh`. Search for `--doctest-modules`.

Reply:
As only a selected few functions are tested in the existing set. I'm a bit confused here. What kind of functions/doc-strings need to be tested?

@TomAugspurger I tried adding the tests in the latest commit, could you take a look?

Reply:
resolving this conversation in favor of more updated discussion below